src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MagicNumberCheck.java

```diff
@@ -212,6 +212,25 @@
  * }
  * </pre>
  *
+ * <p>
+ * Config example of ignoreHashCodeMethod option:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;MagicNumber&quot;&gt;
+ *   &lt;property name=&quot;ignoreHashCodeMethod&quot; value=&quot;true&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * result is no violation:
+ * </p>
+ * <pre>
+ * class TestHashCode {
+ *     public int hashCode() {
+ *         return 10;       // OK
```

Comment:
good, but please add `hashCode` to default example to show that by default it will be violation.
But with  `ignoreHashCodeMethod` - there are no violation.

Reply:
addressed the comment